handlers/profile/workspaces.py

```diff
@@ -0,0 +1,26 @@
+from flask import request, redirect, url_for, abort
+
+from models.workspaces import Workspace
+from utils.decorators import login_required, set_csrf
+from utils.translations import render_template_with_translations
+
+@login_required
+@set_csrf
+def workspaces_list_handler(**params):
+    if request.method == "GET":
+        return render_template_with_translations("profile/main/workspaces.html", **params)
+
+@login_required
+@set_csrf
```

Comment:
@JanVerderber tukaj mora biti `@validate_csrf` (poglej v utils/decorators.py).

CSRF token je način zaščite pred CSRF napadom. Tako da boš moral dati tudi v HTML template v create workspace formo en hidden input, kjer bo notri CSRF token:

```python
<input type="hidden" name="csrf" value="{{ csrf }}">
```